packages/forms-web-app/src/dynamic-forms/dynamic-components/utils/submission-information-utils.js

```diff
@@ -15,6 +15,7 @@ const typeCodeToSubmissionInformationString = {
 exports.formatBeforeYouStartSection = async (appellantSubmission) => {
 	const { LPACode, appealTypeCode, applicationDecisionDate } = appellantSubmission;
 
+	// todo(journey-refactor): api call
```

Comment:
I think we could probably just take this file out and offer a way to push extra data into summaries?

Reply:
not sure we even need to share this right?